core/merge.js

```diff
@@ -156,6 +156,11 @@ class Merge {
     } else if (file.sides.local && file.sides.local >= file.sides.remote) {
       // The file was updated on local after being synched to remote
       return this.updateFileAsync(side, doc)
+    } else if (file.sides.local && file.sides.local < file.sides.remote) {
+      delete doc.remote
+      delete doc.sides
+      metadata.markSide('local', doc)
```

Comment:
I don't think we need to `markSide` before calling `resolveConflictAsync`

Reply:
You're right, we're not saving `doc` into the pouch so it's unnecessary. As are the two `delete` above.